src/settings/target/manifest.rs

```diff
@@ -31,7 +31,7 @@ pub struct Manifest {
     pub workers_dev: Option<bool>,
     #[serde(default = "some_string")]
     pub route: Option<String>,
-    pub routes: Option<HashMap<String, String>>,
```

Comment:
interested in why this was a hashmap to begin with - i know we never used it anywhere but still interested

Reply:
because technically routes are a map between a pattern and a script, and it is potentially useful within a Wrangler project if you wanted to "negate" a route by setting it to a `null` worker. For the purposes of this refactor, I am starting with the simplest implementation - a list of patterns - to reach parity with serverless.

Reply:
makes sense - will this bite us at all when we try to migrate? will the future require breaking changes to the toml? would like to avoid that as much as possible

Reply:
predicting the future of routes is about as easy as predicting next year's weather (probably will be hotter than today, but who knows if it'll be raining). That being said, if it is clear before release that we need this data structure to change we can do that.